README.md

```diff
@@ -83,6 +83,7 @@ For concrete examples of how to integrate OPA with systems like [Kubernetes](htt
 
 ## Presentations
 
+- OPA maintainers talk @ kubecon NA 2022: [video](https://www.youtube.com/watch?v=RMiovzGGCfI)
```

Comment:
kubecon -> KubeCon

Besides that, LGTM!

Reply:
fixed!